`make install` fails on go-cli-build

```console
ENG000656:makeup aaronschlesinger$ make install
Cloning into '.makeup/makeup-bag-deis'...
remote: Counting objects: 12, done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 12 (delta 1), reused 0 (delta 0), pack-reused 5
Unpacking objects: 100% (12/12), done.
Checking connectivity... done.
glide install
[WARN] Unable to locate home directory
[INFO] Fetching updates for github.com/BurntSushi/toml.
[INFO] Fetching updates for github.com/kr/pretty.
[INFO] Fetching updates for github.com/inconshreveable/mousetrap.
[INFO] Fetching updates for github.com/cpuguy83/go-md2man.
[INFO] Fetching updates for github.com/ugorji/go.
[INFO] Fetching updates for github.com/coreos/go-etcd.
[INFO] Fetching updates for github.com/kr/text.
[INFO] Fetching updates for github.com/mitchellh/mapstructure.
[INFO] Fetching updates for github.com/armon/consul-api.
[INFO] Fetching updates for github.com/kr/pty.
[INFO] Fetching updates for github.com/shurcooL/sanitized_anchor_name.
[INFO] Fetching updates for github.com/spf13/cobra.
[INFO] Fetching updates for github.com/spf13/jwalterweatherman.
[INFO] Fetching updates for github.com/spf13/cast.
[INFO] Fetching updates for github.com/spf13/pflag.
[INFO] Fetching updates for github.com/tcnksm/go-gitconfig.
[INFO] Fetching updates for github.com/spf13/viper.
[INFO] Fetching updates for github.com/hashicorp/hcl.
[INFO] Fetching updates for github.com/russross/blackfriday.
[INFO] Fetching updates for github.com/magiconair/properties.
[INFO] Fetching updates for github.com/xordataexchange/crypt.
[INFO] Fetching updates for golang.org/x/crypto.
[INFO] Fetching updates for gopkg.in/fsnotify.v1.
[INFO] Fetching updates for gopkg.in/yaml.v2.
[INFO] Setting version for github.com/armon/consul-api to dcfedd50ed5334f96adee43fc88518a4f095e15c.
[INFO] Setting version for github.com/kr/pty to f7ee69f31298ecbe5d2b349c711e2547a617d398.
[INFO] Setting version for github.com/kr/pretty to e6ac2fc51e89a3249e82157fa0bb7a18ef9dd5bb.
[INFO] Setting version for github.com/kr/text to bb797dc4fb8320488f47bf11de07a733d7233e1f.
[INFO] Setting version for github.com/magiconair/properties to 84ea5e93a95c5e0115f3a4f3e141448584f48f37.
[INFO] Setting version for github.com/BurntSushi/toml to 056c9bc7be7190eaa7715723883caffa5f8fa3e4.
[INFO] Setting version for github.com/russross/blackfriday to d18b67ae0afd61dae240896eae1785f00709aa31.
[INFO] Setting version for github.com/tcnksm/go-gitconfig to 6411ba19847f20afe47f603328d97aaeca6def6f.
[INFO] Setting version for github.com/hashicorp/hcl to 17c5c97b5a51e82c238678127e413b7635c6d5f6.
[INFO] Setting version for github.com/inconshreveable/mousetrap to 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75.
[INFO] Setting version for github.com/mitchellh/mapstructure to 281073eb9eb092240d33ef253c404f1cca550309.
[INFO] Setting version for github.com/spf13/pflag to 08b1a584251b5b62f458943640fc8ebd4d50aaa5.
[INFO] Setting version for github.com/spf13/jwalterweatherman to c2aa07df593850a04644d77bb757d002e517a296.
[INFO] Setting version for github.com/cpuguy83/go-md2man to 71acacd42f85e5e82f70a55327789582a5200a90.
[INFO] Setting version for github.com/shurcooL/sanitized_anchor_name to 10ef21a441db47d8b13ebcc5fd2310f636973c77.
[INFO] Setting version for github.com/ugorji/go to cd43bdd6be4b5675a0d1e75c4af55ee1dc0d9c5e.
[INFO] Setting version for github.com/spf13/cast to ee815aaf958c707ad07547cd62150d973710f747.
[INFO] Setting version for github.com/spf13/viper to 79971f1ae71e74d2e814b9b4bb91fb59b0263654.
[INFO] Setting version for github.com/spf13/cobra to ee6224d01f6a83f543ae90f881b703cf195782ba.
[INFO] Setting version for github.com/coreos/go-etcd to 003851be7bb0694fe3cc457a49529a19388ee7cf.
[INFO] Setting version for github.com/xordataexchange/crypt to 749e360c8f236773f28fc6d3ddfce4a470795227.
[INFO] Setting version for golang.org/x/crypto to f18420efc3b4f8e9f3d51f6bd2476e92c46260e9.
[INFO] Setting version for gopkg.in/yaml.v2 to f7716cbe52baa25d2e9b0d0da546fcf909fc16b4.
[INFO] Setting version for gopkg.in/fsnotify.v1 to 2cdd39bd6129c6a49c74fb07fb9d77ba1271c572.
go build -o bin/makeup -ldflags "-X main.version=devel" main.go
main.go:17:8: cannot find package "github.com/deis/makeup/cmd" in any of:
	/usr/local/go/src/github.com/deis/makeup/cmd (from $GOROOT)
	/Users/aaronschlesinger/gocode/src/github.com/deis/makeup/cmd (from $GOPATH)
make: *** [go-cli-build] Error 1
```

--- a/README.md
+++ b/README.md
@@ -6,7 +6,7 @@ makeup
 Quickstart
 ----------
 
-1. Clone this repository
+1. Clone this repository into `$GOPATH/src/deis`
 
   ```
   git clone https://github.com/sgoings/makeup.git

